neurom/core/__init__.py

```diff
@@ -27,3 +27,8 @@
 # SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 """Core functionality and data types of NeuroM."""
+
+# those imports here for backward compatibility
+from neurom.core.soma import Soma
+from neurom.core.neuron import Section, Neurite, Neuron
```

Comment:
```suggestion
from neurom.core.neuron import (Section, Neurite, Neuron, iter_neurites,
                                iter_sections, iter_segments, graft_neuron)
```

Reply:
shouldn't we keep the same previous API?

Reply:
`iter_neurites, iter_sections, iter_segments, graft_neuron` are part of `neurom/__init__.py`

Reply:
hmmm yeah. But it's still breaking then. I don't know what's the best but in morph-tool for example, you set the requirement as `'neurom>=2.0,<3'` so we could suppose that no change would be needed before `NeuroM==3`.

Reply:
well, it will break but you don't need to change `neurom>=2.0,<3` to `neurom>=2.1,<3`. Just update imports and keep `neurom>=2.0,<3`

Reply:
Yes. It does not comply with semantic versioning but I don't know the BBP policy about that (if there is any?).
Anyway, we can revert only these and update the other codes if you think it's better.

Reply:
Sure, I agree. But also the less break the better :grin: 
But let's go for the less duplication.